README.md

```diff
@@ -23,12 +23,12 @@
 
 ## Source Connectors
 
-| Connector          | Release     | Type | Protocols/Description |
-| :---               | :----:      | :--- | :---                  |
-| [sources/http]     | 0.2.0       | Rust | HTTP 1.0, 1.1, 2.0    |
-| [sources/mqtt]     | 0.1.1       | Rust | MQTT V4, V5           |
-| [sources/postgres] | 0.1.0       | Rust | Postgres CDC          |
-| [sources/syslog]   | 0.1.0       | Rust | Syslog                |
+| Connector          | Release | Type | Protocols/Description      |
+|:-------------------|:-------:|:-----|:---------------------------|
+| [sources/http]     |  0.4.0  | Rust | HTTP Polling 1.0, 1.1, 2.0 |
```

Comment:
Instead of hard coding.  Can we auto generate latest version like here? https://github.com/Ileriayo/markdown-badges

Reply:
It's relatively easy for things on crates.io, but most of these are published as container images. I'm not aware of anything that will do the same for Docker Hub images, since the docker registry API doesn't really allow for it.

Reply:
it looks like we could use: https://shields.io/
<img width="1006" alt="Screen Shot 2022-10-13 at 2 09 38 PM" src="https://user-images.githubusercontent.com/22335041/195687827-f111e108-f006-4c28-a497-2cdbca5dc473.png">

Reply:
Interesting, but we aren't really tagging clean semver versions, at least not without also including a long commit hash. I think we need the release and versioning process to be more reasonable before we try to add ornaments.

Reply:
Fair enough.  Let's use this issue to track that: https://github.com/infinyon/fluvio-connectors/issues/365